apps/site/lib/site_web/controllers/route_controller.ex

```diff
@@ -26,4 +50,57 @@ defmodule SiteWeb.RouteController do
     |> Enum.filter(&(!is_nil(&1.representative_trip_polyline)))
     |> Enum.map(&Polyline.new(&1, color: "#" <> route.color, weight: 4))
   end
+
+  # Use the route mode to determine the display fare. e.g. instead of the 23 bus
+  # showing the free fare, show the bus fare
+  defp display_fare_class(%Route{id: id, fare_class: fare_class} = route)
+       when fare_class not in @display_fare_classes do
+    if Fares.express?(id) do
+      :express_bus_fare
+    else
+      case Route.type_atom(route) do
+        :subway ->
+          :rapid_transit_fare
+
+        :bus ->
+          :local_bus_fare
+
+        :commuter_rail ->
+          :commuter_rail_fare
+
+        :ferry ->
+          :ferry_fare
+
+        # probably a shuttle??
+        _ ->
+          :local_bus_fare
+      end
+    end
+  end
+
+  defp display_fare_class(%Route{fare_class: fare_class}), do: fare_class
+
+  defp price_range(fares) do
+    {min_price, max_price} = Enum.min_max_by(fares, & &1.cents)
+
+    if min_price == max_price do
+      Fares.Format.price(min_price)
+    else
+      "#{Fares.Format.price(min_price)} – #{Fares.Format.price(max_price)}"
+    end
+  end
+
+  defp fare_name(%Fares.Fare{mode: :commuter_rail}) do
+    "Commuter rail one-way"
+  end
+
+  defp fare_name(%Fares.Fare{mode: :ferry}) do
+    "Ferry one-way"
+  end
+
+  defp fare_name(fare) do
+    Fares.Format.name(fare)
+    |> String.capitalize()
+    |> String.replace_suffix("", " one-way")
```

Comment:
I feel like these name functions should also be in the `Fares.Format` module.  Trying to keep everything in one place